ypathgen/pathgen.go

```diff
@@ -173,6 +177,15 @@ type GenConfig struct {
 	// If any key is not a wildcard, then this flag doesn't apply, since
 	// all key values must now be specified in the path.
 	SimplifyWildcardPaths bool
+	// SplitByModule controls whether to generate a go package for each yang module.
+	SplitByModule bool
+	// FakeRootPackageName is the name of package that contains the fakeroot, only
+	// used when splitting generated code by yang module.
+	FakeRootPackageName string
```

Comment:
If the above comment is adopted, then we can drop this option and simply use `GenConfig.PackageName` for the fake root's package name.

Reply:
done